openmetadata-ui/src/main/resources/ui/src/components/SettingsIngestion/SettingsIngestion.component.tsx

```diff
@@ -0,0 +1,308 @@
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { AxiosError } from 'axios';
+import ErrorPlaceHolderIngestion from 'components/common/error-with-placeholder/ErrorPlaceHolderIngestion';
+import Ingestion from 'components/Ingestion/Ingestion.component';
+import Loader from 'components/Loader/Loader';
+import { usePermissionProvider } from 'components/PermissionProvider/PermissionProvider';
+import { OPENMETADATA } from 'constants/Services.constant';
+import { ServiceCategory } from 'enums/service.enum';
+import { PipelineType } from 'generated/api/services/ingestionPipelines/createIngestionPipeline';
+import { DatabaseService } from 'generated/entity/services/databaseService';
+import { IngestionPipeline } from 'generated/entity/services/ingestionPipelines/ingestionPipeline';
+import { Paging } from 'generated/type/paging';
+import { useAirflowStatus } from 'hooks/useAirflowStatus';
+import { ServicesType } from 'interface/service.interface';
+import { isUndefined } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  deleteIngestionPipelineById,
+  deployIngestionPipelineById,
+  enableDisableIngestionPipelineById,
+  getIngestionPipelines,
+  triggerIngestionPipelineById,
+} from 'rest/ingestionPipelineAPI';
+import { fetchAirflowConfig } from 'rest/miscAPI';
+import { getServiceByFQN } from 'rest/serviceAPI';
+import { showErrorToast } from 'utils/ToastUtils';
+import { SettingsIngestionProps } from './SettingsIngestion.interface';
+
+function SettingsIngestion({
+  pipelineType,
+  handleIngestionDataChange,
+  handleIngestionPipelinesChange,
+  handleServiceDetailsChange,
+}: SettingsIngestionProps) {
+  const { t } = useTranslation();
+  const { permissions } = usePermissionProvider();
+  const { isAirflowAvailable } = useAirflowStatus();
+  const [isLoading, setIsLoading] = useState(true);
+  const [serviceList] = useState<Array<DatabaseService>>([]);
+  const [ingestionPipelines, setIngestionPipelines] = useState<
+    IngestionPipeline[]
+  >([]);
+  const [serviceDetails, setServiceDetails] = useState<ServicesType>();
+  const [airflowEndpoint, setAirflowEndpoint] = useState<string>();
+  const [ingestionPaging, setIngestionPaging] = useState<Paging>({} as Paging);
+  const [ingestionCurrentPage, setIngestionCurrentPage] = useState(1);
+
+  const serviceCategory = ServiceCategory.METADATA_SERVICES;
+  const serviceFQN = OPENMETADATA;
+
+  const getAirflowEndpoint = async () => {
+    try {
+      setIsLoading(true);
+      const res = await fetchAirflowConfig();
+      setAirflowEndpoint(res.apiEndpoint);
```

Comment:
I feel we have airflow hook we can use that https://github.com/open-metadata/OpenMetadata/blob/cfacbfa16813b614bd2f9dd218d03caf60e7a539/openmetadata-ui/src/main/resources/ui/src/hooks/useAirflowStatus.ts

Reply:
will take a look

Reply:
@chirag-madlani the hook that we have is to check the airflow status with a different API call. Here we are fetching the airflowEndpoint